### PR TITLE
[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute

### PR DESCRIPTION
**Pull request by alexbuckgit**
[BULK UPDATE] DocuTune - Fix build validation issues: docs-link-absolute

**Global effort to fix build validation errors**

The Microsoft Skilling team is fixing build validation errors and specific brand inconsistencies in Microsoft technical documentation for the rest of FY23 Q1. This will help address various accessibility, security, and usability issues. This PR was generated using DocuTune. It includes only targeted fixes and minor formatting adjustments.

**If this content is generated from another source, please make the corresponding changes in the source location.**

DocuTune PRs improve quality of technical content by finding and automatically correcting a broad set of issues only where the correction is suitable. Please review within five business days and merge or comment in the PR with any changes you'd like to see. Thank you!

CorrelationId: 0df9c2d6-63f8-4d1e-8b57-b33611a06514
InitiativeId: docutune-build-validation-issues-2022-05
PointOfContact: Alex Buck

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

File: docs-ref-autogen/azure-devops-extension-api/GitRestClient.yml [hunk 1]
```diff
@@ -219,9 +219,8 @@ methods:
     package: azure-devops-extension-api
     summary: >-
       Request that another repository's refs be fetched into this one. It syncs
-      two existing forks. To create a fork, please see the \<a
-      href="https://docs.microsoft.com/en-us/rest/api/vsts/git/repositories/create?view=azure-devops-rest-5.1"\>
-      repositories endpoint\</a\>
+      two existing forks. To create a fork, please see the
+      [repositories endpoint](/rest/api/vsts/git/repositories/create?view=azure-devops-rest-5.1).
```
Comment (jmart1428) [inline docs-ref-autogen/azure-devops-extension-api/GitRestClient.yml]: This link works, but both the original link and the edited link redirect to version 5.0 even though version 5.1 is available.

**Comment by jmart1428** [comment]
#docutune-wait

File: docs-ref-autogen/azure-devops-extension-api/GitRestClient.yml (hunk 1 above)
Comment (alexbuckgit) [reply docs-ref-autogen/azure-devops-extension-api/GitRestClient.yml]: Updated to use the redirected link.

**Comment by alexbuckgit** [comment]
#docutune-ready

**Comment by alexbuckgit** [comment]
@yishengjin1413 @sudeepku

Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

#docutune-request-approval

**Comment by alexbuckgit** [comment]
@AngryBerryMS @herohua @jeffreyxrh @kcpitt @kenanlv @mikekinsman @rloutlaw @sudeepku @supernova-eng @v-caxian @v-pegao @VSC-Service-Account @yishengjin1413 @yunair
 
 Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

**Comment by alexbuckgit** [comment]
@nechvatalp

Please merge the changes in this PR. For more information, see the description at the top of this PR.

Any pre-existing build validation issues or insufficient Acrolinx scores should be addressed in a separate PR.

**Pull request merged**